tests/cli/noArgsDeploy.test.ts

```diff
@@ -0,0 +1,202 @@
+
+  it('should create a starknet account', async () => {
+    if (fs.existsSync(path.resolve(__dirname, 'starknet_open_zeppelin_accounts.json'))) {
+      fs.unlinkSync(path.resolve(__dirname, 'starknet_open_zeppelin_accounts.json'));
+    }
+
+    const { stdout, stderr } = await sh(
+      `${warpBin} new_account --wallet ${wallet} --network ${network} --feeder_gateway_url ${gatewayURL} --account_dir ${starkNetAccountDir} --account account_0`,
+    );
+
+    expect(stderr).to.be.empty;
+    expect(stdout).to.not.be.empty;
+
+    const starknet_open_zeppelin_accounts = JSON.parse(
+      fs.readFileSync(path.resolve(__dirname, 'starknet_open_zeppelin_accounts.json'), 'utf-8'),
+    );
+
+    // verify the properties of the created account
+    expect(starknet_open_zeppelin_accounts[network]).to.not.be.undefined;
+    expect(starknet_open_zeppelin_accounts[network]['account_0']).to.not.be.undefined;
+    expect(starknet_open_zeppelin_accounts[network]['account_0']['private_key']).to.not.be
+      .undefined;
+    expect(starknet_open_zeppelin_accounts[network]['account_0']['public_key']).to.not.be.undefined;
+    expect(starknet_open_zeppelin_accounts[network]['account_0']['salt']).to.not.be.undefined;
+    expect(starknet_open_zeppelin_accounts[network]['account_0']['address']).to.not.be.undefined;
+    expect(starknet_open_zeppelin_accounts[network]['account_0']['deployed']).to.be.false;
+  });
+
+  it('mint wei to the generated account', async () => {
+    const starknet_open_zeppelin_accounts = JSON.parse(
+      fs.readFileSync(path.resolve(__dirname, 'starknet_open_zeppelin_accounts.json'), 'utf-8'),
+    );
+    const address = starknet_open_zeppelin_accounts[network]['account_0']['address'];
+
+    const { stdout } = await mintEthToAccount(address);
+
+    const response = JSON.parse(stdout);
+
+    expect(response).to.not.be.undefined;
+    expect(response['new_balance']).to.be.equal(1000000000000000000);
+    expect(response['tx_hash']).to.not.be.undefined;
+  });
+
+  it('should deploy the starknet account', async () => {
+    const { stdout, stderr } = await sh(
+      `${warpBin} deploy_account --wallet ${wallet} --feeder_gateway_url ${gatewayURL} --gateway_url ${gatewayURL} --network ${network} --account_dir ${starkNetAccountDir} --account account_0`,
+    );
+
+    expect(stderr).to.be.empty;
+    expect(stdout).to.not.be.empty;
+
+    const txFeeEth = extractFromStdout(stdout, TX_FEE_ETH_REGEX);
+    const txFeeWei = extractFromStdout(stdout, TX_FEE_WEI_REGEX);
+    const contract_address = extractFromStdout(stdout, CONTRACT_ADDRESS_REGEX);
+    const txHash = extractFromStdout(stdout, TX_HASH_REGEX);
+
+    expect(txFeeEth).to.not.be.undefined;
+    expect(txFeeWei).to.not.be.undefined;
+    expect(contract_address).to.not.be.undefined;
+    expect(txHash).to.not.be.undefined;
+
+    const res = await sh(
+      `${warpBin} status ${txHash} --network ${network} --gateway_url ${gatewayURL} --feeder_gateway_url ${gatewayURL}`,
```

Comment:
Imo a function `runWarpWithStarknetDefaults` or a string `STARKNET_DEFAULT_OPTIONS` would improve the readability of shell commands

Reply:
abstracted the network options and the account options so its a lot cleaner now